src/train.js

```diff
@@ -1,3 +1,5 @@
+let g_progressBar, g_coeffs_div, g_accuracy_div, g_goodBadDiv;
```

Comment:
Let's make these names consistent (`_div` vs. `Div`). Everything else is camelCase. Shall we stick with that? I wouldn't even mind `gProgressBar`, though stick with g_ if you have a strong opinion.

Reply:
FWIW, I also feel the interest on the technical debt of not using jquery or React or something every time I have to write raw DOM code. :-/